tests/benchmarks/data_structure_perf/dlist_perf/src/dlist_perf.c

```diff
@@ -254,14 +254,4 @@ void test_dlist_insert_and_remove(void)
 		&node[ARRAY_SIZE(node)/2 - 1]), "dlist can't remove a node in constant time");
 }
 
-/* ztest main entry */
-void test_main(void)
-{
-	ztest_test_suite(test_dlist,
-			ztest_unit_test(test_dlist_container),
```

Comment:
need to convert `test_dlist_container`

Reply:
Oh, I missed this. Thanks for your reviewing! :)